etl/steps/data/garden/un/2023-08-29/long_run_child_mortality.py

```diff
@@ -0,0 +1,88 @@
+"""Load a meadow dataset and create a garden dataset."""
+import owid.catalog.processing as pr
+from owid.catalog import Table
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_igme = paths.load_dataset("igme")
+    ds_gapminder = paths.load_dataset("under_five_mortality")
+    # Read table from meadow dataset.
+    tb_igme = ds_igme["igme"].reset_index()
+
+    # Select out columns of interest.
+    cols = [
+        "country",
+        "year",
+        "observation_value_deaths_per_1_000_live_births_under_five_mortality_rate_both_sexes_all_wealth_quintiles",
+    ]
+    tb_igme = tb_igme[cols]
+    tb_igme = tb_igme.rename(
+        columns={
+            "observation_value_deaths_per_1_000_live_births_under_five_mortality_rate_both_sexes_all_wealth_quintiles": "under_five_mortality"
+        }
+    )
+
+    tb_igme["source"] = "igme"
+    # Load full Gapminder data
+    tb_gap = ds_gapminder["under_five_mortality"].reset_index()
+    tb_gap["source"] = "gapminder"
+    #
+    tb_gap_sel = ds_gapminder["under_five_mortality_selected"].reset_index()
+    tb_gap_sel["source"] = "gapminder"
+
+    # Combine IGME and Gapminder data
+    tb_combined = combine_datasets(tb_igme, tb_gap, "long_run_child_mortality")
+    tb_combined_sel = combine_datasets(tb_igme, tb_gap_sel, "long_run_child_mortality_selected")
+
+    #
+    # Save outputs.
+    tb_combined = tb_combined.drop(columns=["source"]).set_index(["country", "year"], verify_integrity=True)
+    tb_combined_sel = tb_combined_sel.drop(columns=["source"]).set_index(["country", "year"], verify_integrity=True)
+    #
+    # Create a new garden dataset with the same metadata as the meadow dataset.
+    ds_garden = create_dataset(dest_dir, tables=[tb_combined, tb_combined_sel], check_variables_metadata=True)
+    ds_garden.update_metadata(metadata_path=paths.metadata_path)
```

Comment:
I don't think you need to use update_metadata. This should happen automatically inside `create_dataset`.